app/src/main/java/com/example/sharingang/CacheDatabase.kt

```diff
@@ -8,7 +8,7 @@ import com.example.sharingang.items.ItemDao
 import com.example.sharingang.users.User
 import com.example.sharingang.users.UserDao
 
-@Database(entities = [Item::class, User::class], version = 1)
+@Database(entities = [Item::class, User::class], version = 1, exportSchema = false)
```

Comment:
What does `exportSchema = false`  do?

Reply:
If set to true, it would generate files like this:
```json
{
  "formatVersion": 1,
  "database": {
    "version": 1,
    "identityHash": "53db508c5248423325bd5393a1c88c03",
    "entities": [
      {
        "tableName": "item_table",
        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`id` INTEGER PRIMARY KEY AUTOINCREMENT, `title` TEXT)",
        "fields": [
          {
            "fieldPath": "id",
            "columnName": "id",
            "affinity": "INTEGER"
          },
          {
            "fieldPath": "title",
            "columnName": "title",
            "affinity": "TEXT"
          }
        ],
        "primaryKey": {
          "columnNames": [
            "id"
          ],
          "autoGenerate": true
        },
        "indices": [],
        "foreignKeys": []
      }
    ],
    "setupQueries": [
      "CREATE TABLE IF NOT EXISTS room_master_table (id INTEGER PRIMARY KEY,identity_hash TEXT)",
      "INSERT OR REPLACE INTO room_master_table (id,identity_hash) VALUES(42, \"53db508c5248423325bd5393a1c88c03\")"
    ]
  }
}
```
It could be useful to track changes to the database schema when most of the development is finished and the properties of Item/User are no longer changing.

Reply:
Oh, I see, thank you for the clarification!